client/my-sites/checkout/composite-checkout/types/country-list-item.ts

```diff
@@ -1,4 +1,5 @@
 export interface CountryListItem {
 	code: string;
 	name: string;
+	has_postal_codes: boolean;
```

Comment:
Where does this property get set? Looking at the `state.countries.domains` and `state.countries.payments` records, I don't see it set for `CW`:

<img width="374" alt="Screen Shot 2021-06-15 at 6 02 09 PM" src="https://user-images.githubusercontent.com/2036909/122129569-4ce1f580-ce04-11eb-8915-4a7abfc78ab6.png">

Reply:
>Is there a server-side patch that's required to make this data available?

Yes! There is. I also added a patch for the countries API, where it would also return the `has_postal_code` property, but I wasn't really sure whether to create the PR before applying it, so I did... 
If that's not the case then please let me know. 😅